python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_postprocessing.py

```diff
@@ -280,7 +276,7 @@ def _summarize(
 
     @dbt_assets(manifest=test_jaffle_shop_manifest_standalone_duckdb_dbfile)
     def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
-        yield from dbt.cli(["build"], context=context).stream()._attach_metadata(_summarize)  # noqa: SLF001
+        yield from dbt.cli(["build"], context=context).stream().attach_metadata(_summarize)
```

Comment:
`.attach_metadata` feels weird, considering we already have `.with_metadata` on the event. Could we standardize on the action here?

Reply:
maybe it should be `map_<something>` to communicate that it applies a function? `attach_metadata` makes me expect it would accept a metadata dict